spark/src/test/scala/org/apache/comet/exec/CometNativeShuffleSuite.scala

```diff
@@ -201,6 +202,23 @@ class CometNativeShuffleSuite extends CometTestBase with AdaptiveSparkPlanHelper
     }
   }
 
+  test("fix: Comet native shuffle deletes shuffle files after query") {
+    withParquetTable((0 until 5).map(i => (i, i + 1)), "tbl") {
+      sql("SELECT count(_2), sum(_2) FROM tbl GROUP BY _1").collect()
+      val diskBlockManager = SparkEnv.get.blockManager.diskBlockManager
+      var hasShuffleFiles = true
+      var counter = 0
+      while (hasShuffleFiles && counter < 30) {
```

Comment:
nit: use [eventually](https://www.scalatest.org/scaladoc/3.2.16/org/scalatest/concurrent/Eventually.html), like:

```
eventually(timeout(30.seconds), interval(1.seconds)) {
  System.gc()
  assert(diskBlockManager.getAllFiles().isEmpty)
}
```

Reply:
That's neat! Switched to eventually in https://github.com/apache/datafusion-comet/pull/1568/commits/8855913b56280b602215136cd3367d509318a11e.